__tests__/news.test.js

```diff
@@ -0,0 +1,97 @@
+const app = require("../app.js");
+const request = require("supertest");
+const testData = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    return db.end();
+});
+
+describe("GET /api/topics", () => {
+
+    it("Should return an array of users objects", () => {
+
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then((response) => {
+                response.body.topics.forEach((topic) => {
+
+                    expect(topic).toHaveProperty("slug", expect.any(String));
+                    expect(topic).toHaveProperty("description", expect.any(String));
+
+                });
```

Comment:
whenever we are iterating through an array in our tests we need to ensure the array is not empty!

You need to make sure `response.body.topics.length` is not `0`!